bin/run_cli

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
+java -jar build/libs/TicTacToe-all.jar  cli
```

Comment:
Don't you want to build the thing first?

Can you do the `$SCRIPT_DIR` thing in the path, so that this works regardless of the current working directory?